ballerina/Module.md

```diff
@@ -1,17 +1,219 @@
+<img src=https://github.com/user-attachments/assets/1c565923-e968-4d5f-9864-7ed2022b8079 alt="OpenAI Platform" style="width: 70%;">
+
+
+5. Store the API key securely to use in your application 
+<img src=https://github.com/user-attachments/assets/bbbf8f38-d551-40ee-9664-f4cf2bd98997 alt="OpenAI Platform" style="width: 70%;">
+
+
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+A typical integration of the Assistants API has the following flow:
+
+1. **Create an Assistant**
+    - Define its custom instructions and pick a model.
+    - If helpful, add files and enable tools like Code Interpreter, File Search, and Function calling.
+
+2. **Create a Thread**
+    - Create a Thread when a user starts a conversation.
+
+3. **Add Messages to the Thread**
+    - Add Messages to the Thread as the user asks questions.
+
+4. **Run the Assistant**
+    - Run the Assistant on the Thread to generate a response by calling the model and the tools.
+
+This starter guide walks through the key steps to create and run an Assistant that uses the Code Interpreter tool. In this example, we're creating an Assistant that is a personal math tutor.
+### Setting HTTP Headers in Ballerina
+
+Calls to the Assistants API require that you pass a beta HTTP header. In Ballerina, you can define the header as follows:
+
+```ballerina
+final map<string|string[]> headers = {
+    "OpenAI-Beta": ["assistants=v2"]
+};
+```
+
+### Step 1 : Setting up the connector
+To use the `OpenAI Assistants` connector in your Ballerina application, update the `.bal` file as follows:
+
+1. Import the `openai_assistants` module.
+
+```ballerina
+import ballerinax/openai_assistants;
+```
+
+2. Create a `Config.toml` file and configure the obtained credentials as follows:
+
+```bash
+token = "<Access Token>"
+```
+
+3. Create a `openai_assistants:Client` with the obtained access token and initialize the connector with it.
+
+```ballerina
+configurable string token = ?;
+
+final openai_assistants:Client AssistantClient = check new({
+    auth: {
+        token
+    }
+});
+```
+
+### Step 2: Create an Assistant
+
+Now, utilize the available connector operations to create an Assistant.
+
+
+
+```ballerina
+public function main() returns error? {
+
+    // define the required tool
+    openai_assistants:AssistantToolsCode tool = {
+        type: "code_interpreter"
+    };
+
+    // define the assistant request object
+    openai_assistants:CreateAssistantRequest request = {
+        model: "gpt-3.5-turbo",
+        name: "Math Tutor",
+        description: "An Assistant for personal math tutoring",
+        instructions: "You are a personal math tutor. Help the user with their math questions.",
+        tools: [tool]
+    };
+
+    // Call the `post assistants` resource to create an Assistant
+   var response = check AssistantClient->/assistants.post(request, headers);
```

Comment:
lets use the proper record types instead of using `var`
```suggestion
   var response = check AssistantClient->/assistants.post(request, headers);
```

Reply:
please fix all the similar places